model/user-agent/registry.yaml

```diff
@@ -35,3 +35,20 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+      - id: user_agent.synthetic.type
```

Comment:
Asked this in chat, but also asking here.

Should this (also) be added to client spans so synthetic agents can self-identify in a trace?

e.g. https://opentelemetry.io/blog/2023/synthetic-testing/

Reply:
Makes sense to me to allow agents to self-identify and allow for propagation of `user_agent.synthetic.type` to any server spans created in response to the remote client span from the synthetic agent.

Reply:
these sound like two different (though potentially both useful) things:

> allow agents to self-identify

and

> allow for propagation of `user_agent.synthetic.type` to any server spans created in response to the remote client span from the synthetic agent

I'd suggest sticking to just the first in this PR

Reply:
I agree that both are useful. I'd be happy to keep this PR focused for specific review of the `synthetic.type` on server spans. @jsuereth does limiting the scope here work for you?

Reply:
@jsuereth per our discussion in the semantic convention WG, I've added further information regarding the usefulness of this attribute for client self-identification as synthetic.